src/iexamine.cpp

```diff
@@ -3530,29 +3574,28 @@ void iexamine::autodoc( player &p, const tripoint &examp )
             const itype *itemtype = it->type;
             const bionic_id &bid = itemtype->bionic->id;
 
-            if( p.has_bionic( bid ) ) {
-                popup( _( "You have already installed this bionic."  ) );
+	    installer.add_msg_player_or_npc(
+                _( "You prepare to install the %1$s on %2$s." ),
+                _( "<npcname> prepares to install the %1$s on %2$s." ),
```

Comment:
This line implies that NPCs are able to install bionics on other patients. But as I understand it, this PR allows only player characters to install CBMs on NPCs.